README.md

```diff
@@ -1 +1,6 @@
 # Omnipay School EasyPay
+
+This package appears to have been adapted from omnipay-westpac-payway-rest.  It contains a number of methods referencing 
+properties that are particular to that package, and are probably not needed here.
+
+We should do a thorough review of the code to ensure any unnecessary methods are removed.
```

Comment:
Should we make this into a ticket? If its here, it will never get done.

Reply:
Done - https://digistorm.atlassian.net/browse/PSYS-129